src/Theming/Palettes.ts

```diff
@@ -123,7 +123,7 @@ export const fluentDarkThemePalette: IExtendedPartialPalette = {
     neutralLight: '#0c0e0d',
     neutralQuaternaryAlt: '#0b0d0c',
     neutralQuaternary: '#0b0c0c',
-    neutralTertiaryAlt: '#0a0c0b',
+    neutralTertiaryAlt: '#5e5e5e',
```

Comment:
assuming you checked the color code of the same variable in other theme palettes (fluentExplorerThemePalette etc.), they seem all good, right?

Reply:
Yeah, the other themes all looked very similar to the button for adding a map value when the key is blank and the button disabled. Which is the hex value I pulled for here.